Makefile

```diff
@@ -20,3 +20,11 @@ copy-travis-files:
 .PHONY:
 test:
 	bundle exec rubocop -R -D
+
+.PHONY:
```

Comment:
Huh! So has make been creating secret shadow files all along or something?
The command still works even though phony is incorrect.